2-copy-of-code/lesson-18/lesson18.html

```diff
@@ -50,9 +50,14 @@
       */
 
       async function getAmazon() {
-        const response = await fetch('https://amazon.com');
-        const text = await response.text();
-        console.log(text);
+        try {
+          const response = await fetch('https://amazon.com');
+          const text = await response.text();
+          console.log(text);
+
+        } catch (error) {
+          console.log('CORS error. You request was blocked by the backend.');
```

Comment:
Simple English mistake here :) "CORS error. You request..." => "CORS error. Your request..."